community/posters.md

```diff
@@ -0,0 +1,58 @@
+# Event Poster Guidelines
+
+### Poster Design Guidelines
+#### Size
+- Instagram restricts images in a post to 1:1 ratio.
+- For printable posters, the size must be **11.5x17.5 inches** and it must be in portrait orientation.
+
+#### Content
+See also: [kossiitkgp/design](https://github.com/kossiitkgp/design/).
+
+- The name "Kharagpur Open Source Society" and a logo must be mentioned in a prominent position.
+- Event posters:
+	- Each poster must include a date, time, and location of the event.
+	- Contact numbers of at least two people in-charge must be included on the poster.
+- Posters that will be stuck on campus must include a QR code linking to a place where more information could be obtained or registration process could be done. The link could be to a website, a document, or a form.
+- If the poster is about an event that has a sponsor, the poster must be first approved by the sponsor and must adhere to their brand guidelines.
+- The design for the poster with in both an editable format (could be an online link) and an exported image format must be added to the [design](https://github.com/kossiitkgp/design/) repository.
+
+### Poster Printing Guidelines
+- Shops at IIT Kharagpur that print posters:
+	- Quest Printers, Tech Market.
+- The poster must be printed in A3 portrait size. See [#Size](#size).
```

Comment:
There is no `#size`.. yet..

Reply:
Except there is?

Reply:
Doesn't seem to work :(
I get HTML page of this github page instead

Reply:
The preview feature doesn't work perfectly. If you open the doc and click on it, the link works.